skimage/feature/tests/test_peak.py

```diff
@@ -228,6 +228,15 @@ def test_empty(self):
                                      indices=False, exclude_border=False)
         assert np.all(~ result)
 
+    def test_empty_non2d_indices(self):
+        image = np.zeros((10, 10, 10))
+        result = peak.peak_local_max(image,
+                                     footprint=np.ones((3, 3), bool),
+                                     min_distance=1, threshold_rel=0,
+                                     indices=True, exclude_border=False)
+        assert result.shape[0] == 0
```

Comment:
May be use a single check:
```suggestion
        assert result.shape == (0, image.ndim)
```

Reply:
applied.